CONTRIBUTING.md

```diff
@@ -158,10 +158,7 @@ There are three ways to setup an Apache Airflow development environment.
   Start a docker container through Compose for development to avoid installing the packages directly on your system. The following will give you a shell inside a container, run all required service containers (MySQL, PostgresSQL, krb5 and so on) and install all the dependencies:
 
   ```bash
-  docker-compose -f scripts/ci/docker-compose.yml run airflow-testing bash
-  # From the container
-  export TOX_ENV=py35-backend_mysql-env_docker
-  /app/scripts/ci/run-ci.sh
+  docker-compose -f scripts/ci/docker-compose.yml run -e TOX_ENV=py35-backend_mysql-env_docker airflow-testing /app/scripts/ci/run-ci.sh
```

Comment:
Please don't make un-related changes to other files.

Reply:
Is there a specific action I can take in response to this?  Are you asking me to separate this into a different pull request?  Or are you just asking why I changed it?

If the latter, it's because as a new contributor, I used the script line further down in CONTRIBUTING.md to try to run tests, but it did not work.  These lines: https://github.com/apache/airflow/pull/5297/files#diff-6a3371457528722a734f3c51d9238c13R195

Reply:
Please make a separate PR

Reply:
Done: https://github.com/apache/airflow/pull/5412/files